tests/app-definitions/TestApplicationString-output.json

```diff
@@ -44,5 +40,9 @@
   "instances": 2,
   "mem": 64,
   "ports": null,
-  "dependencies": null
+  "dependencies": null,
+  "env": {
```

Comment:
It's marshaled in the order it appears in the struct.  Since the env and secrets are last now, they need to be last in the test output or they won't test as equal.